Builder/FormContractor.php

```diff
@@ -65,12 +65,27 @@ public function fixFieldDescription(AdminInterface $admin, FieldDescriptionInter
         $fieldDescription->setAdmin($admin);
         $fieldDescription->setOption('edit', $fieldDescription->getOption('edit', 'standard'));
 
-        if (in_array($fieldDescription->getMappingType(), array(ClassMetadataInfo::ONE_TO_MANY, ClassMetadataInfo::MANY_TO_MANY, ClassMetadataInfo::MANY_TO_ONE, ClassMetadataInfo::ONE_TO_ONE))) {
+        if ($this->hasAssociation($fieldDescription) || $fieldDescription->getOption('admin_code')) {
```

Comment:
If somebody add `admin_code` option it should allways attach admin class, right ?